src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/CngKeyHandleOpenOptions.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    ///     Parameter to CngKey.Open(SafeNCryptKeyHandle,...). This is not a mapping of a native NCrypt value. 
```

Comment:
I don't know if we'll ever have MSDN documentation generated from in-source comments, but "This is not a mapping of a native NCrypt value. " feels more like a `// Note: This is NOT a mapping of a native NCrypt value.` comment than a `/// <summary />` one.